pages/auth/index.tsx

```diff
@@ -137,209 +128,156 @@ export default function AuthPage() {
 
   return (
     <>
-      <div className="p-4">
-        <Link href="/" passHref>
-          <div className="cursor-pointer items-center inline-flex">
-            <ChevronLeftIcon />
-            Return to event site
-          </div>
-        </Link>
-      </div>
-      <section className="py-2 md:px-16 px-10 flex lg:justify-between justify-center flex-wrap">
-        <div className="xl:w-1/2 lg:w-2/3 w-5/6 my-4">
-          <h1 className="md:text-3xl text-2xl font-black">HackUTD VIII Hacker Registration</h1>
-          <p className="md:text-base text-sm">
-            To complete your application or access event features, please create an account or log
-            in with an existing one.
-          </p>
-          <div className="mt-16 flex text-2xl">
-            <div
-              id="signInOption"
-              className="py-2 mr-6 underline cursor-pointer"
-              onClick={() => changeOption(true)}
-            >
-              Sign In
+      <section className="bg-secondary min-h-screen">
+        <div className="p-4">
+          <Link href="/" passHref>
+            <div className="cursor-pointer items-center inline-flex text-primaryDark font-medium">
+              <ChevronLeftIcon />
+              return to event site
             </div>
-            <div
-              id="signUpOption"
-              className="py-2 cursor-pointer"
-              onClick={() => changeOption(false)}
+          </Link>
+        </div>
+        <div className="py-2 md:px-16 px-10 flex items-center justify-center flex-wrap">
+          <div className="xl:w-1/2 lg:w-2/3 w-5/6 my-4">
+            <section
+              id="signInSection"
+              className="bg-white 2xl:min-h-[30rem] min-h-[28rem] rounded-lg p-6"
             >
-              Create Account
-            </div>
-          </div>
-          <section
-            id="signInSection"
-            className="bg-[#F2F3FF] 2xl:min-h-[30rem] min-h-[28rem] rounded-lg p-6"
-          >
-            {!passwordResetDialog ? (
-              <React.Fragment>
-                <form onSubmit={handleSubmit}>
-                  <h1 className="text-xl font-bold mt-4 mb-2">Email</h1>
+              {!passwordResetDialog ? (
+                <>
+                  {' '}
+                  <h1 className="md:text-3xl text-2xl font-black text-center text-primaryDark mt-4">
```

Comment:
wtf am i looking at here

Reply:
oops

Reply:
sorry abt that! need to learn how to review my own code properly before committing.